package.json

```diff
@@ -34,6 +34,19 @@
     "lint",
     "test"
   ],
+  "browserslist": [
+    "Chrome >= 49",
+    "Firefox >= 48",
+    "Safari >= 6",
+    "Edge >= 12",
+    "IE >= 9",
+    "Opera 47-48",
+
+    "ChromeAndroid >= 38",
+    "ios_saf >= 7",
+    "ie_mob >= 10",
+    "Android >= 3"
+  ],
```

Comment:
Eventually this should come from a global require, correct? We store it in the @ebay org? https://github.com/browserslist/browserslist#shareable-configs

/cc @senthilp

Reply:
Yes, that is the plan. For now, this should be ok.

Reply:
I've created a task to track and prioritize it: https://github.com/eBay/ebayui-core/issues/76.